dist/src/conf/logback.xml

```diff
@@ -37,6 +37,7 @@
   <logger name="com.github.benmanes.caffeine.cache" level="ERROR" />
   <logger name="com.linecorp" level="INFO" />
   <logger name="com.linecorp.centraldogma" level="DEBUG" />
+  <logger name="com.linecorp.centraldogma.server.internal.mirror" level="INFO" />
```

Comment:
I found out that `GitSshdSessionFactory` from Apache MINA, which is Inherited by the `DefaultGitSshdSessionFactory` Central Dogma class, produces verbose debug outputs.
`GitSshdSessionFactory` creates the logger with its class name which is `DefaultGitSshdSessionFactory` in this case so they are all logged by the `<logger name="com.linecorp.centraldogma" level="DEBUG" />`.
```
2023-08-26 00:38:06.468 [DEBUG](c.l.c.s.i.m.GitWithAuth$DefaultGitSshdSessionFactory$1) [mirroring-worker-6-1] Connecting to ...
2023-08-26 00:38:06.537 [DEBUG](c.l.c.s.i.m.GitWithAuth$DefaultGitSshdSessionFactory$1) [mirroring-worker-6-1] Connected to ...
2023-08-26 00:38:06.537 [DEBUG](c.l.c.s.i.m.GitWithAuth$DefaultGitSshdSessionFactory$1) [mirroring-worker-6-1] Authenticating: ClientSessionImpl[git@...]
.....
6-1] Stopped SshClient[3c45db89]
```

Reply:
Can we extract the class (`...$1`) into a top-level class and adjust the log level for that specific class only?

Reply:
There are two separate `DefaultGitSshdSessionFactory` anonymous classes in the file.
Let me just use the class specifically:
` <logger name="com.linecorp.centraldogma.server.internal.mirror.GitWithAuth$DefaultGitSshdSessionFactory" level="INFO" />`

Reply:
SGTM